_targets.R

```diff
@@ -21,11 +21,11 @@ dir.create("3_visualize/out/nhdv2_attr_png/",showWarnings = FALSE)
 
 # Define columns of interest for harmonized WQP data
 wqp_vars_select <- c("MonitoringLocationIdentifier","MonitoringLocationName","LongitudeMeasure","LatitudeMeasure",
-                     "MonitoringLocationTypeName","OrganizationIdentifier","ActivityStartDate","ActivityStartTime.Time",
-                     "ActivityEndDate","CharacteristicName","param_group","param","USGSPCode","ActivityMediaName",
-                     "ResultSampleFractionText","HydrologicCondition","HydrologicEvent","resultVal2","resultUnits2",
-                     "ResultDetectionConditionText","ResultTemperatureBasisText","PrecisionValue","ResultStatusIdentifier",
-                     "final")
+                     "MonitoringLocationTypeName","OrganizationIdentifier","ActivityStartDate","ActivityStartDateTime",
+                     "ActivityStartTime.Time","ActivityStartTime.TimeZoneCode","CharacteristicName","param_group","param",
```

Comment:
```suggestion
                     "ActivityStartTime.Time","ActivityStartTime.TimeZoneCode","ActivityEndDate","CharacteristicName","param_group","param",
```

Reply:
Did you mean to drop the end date?

Reply:
I _did_ intentionally drop the end date here, although I'm also happy to keep it. `ActivityEndDate` is `NA` for the vast majority of discrete samples. I had originally kept this column for QC purposes (_does end date equal start date?_), but for these sites it's hard to interpret the meaning if the end date is different, and so we're not really using it. From what I could tell the temperature [repo](https://github.com/USGS-R/2wp-temp-observations/blob/main/1_wqp_pull/src/get_wqp_data.R) doesn't retain that column, so I omitted it here.

Reply:
okay, dropping `ActivityEndDate` sounds good!